src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -179,6 +181,24 @@ def _download_symbolic_link(dxid, md5digest, project, dest_filename):
                                             duration=6*3600,
                                             project=project)
 
+    def call_cmd(cmd, max_retries=2, num_attempts=0):
```

Comment:
Should you define constant values as DEFAULT_MAX_RETRIED and DEFAULT_NUM_ATTEMPS instead of hardcoding them?  

I don't know too much about engineering practice.  @commandlinegirl

Reply:
Thanks @XingGemini  num_attempts is recursively updated value of the current attempt number and initialized at 0 so it's very local to this function. max_retries could be extracted but it's also rather local to this particular inner function so IMHO can stay as is.

Reply:
Fair enough.  Thanks for the clarification.